vmm/src/cpu.rs

```diff
@@ -74,6 +80,18 @@ use vm_migration::{
 use vmm_sys_util::eventfd::EventFd;
 use vmm_sys_util::signal::{register_signal_handler, SIGRTMIN};
 
+#[cfg(all(target_arch = "aarch64", feature = "gdb"))]
+/// Extract the specified bits of a 64-bit integer.
+/// For example, to extrace 2 bits from offset 1 (zero based) of `6u64`,
```

Comment:
Extract??

Reply:
The wording or the macro's name is not proper, right? Any suggestion?

Reply:
I think using extract should be fine.